src/js/Controllers/DisplayCapabilities.js

```diff
@@ -5,185 +5,92 @@ let softButtonCapability = {
 	"imageSupported": true,
 	"textSupported": true
 }
+
+let imageButtonCapability = softButtonCapability
+imageButtonCapability.textSupported = false
```

Comment:
this line also changes the softButtonCapability object 😬 
everything in JS is a reference so I would just make a new object

Reply:
Oh well, so much for that slight improvement, fixed in https://github.com/smartdevicelink/generic_hmi/pull/248/commits/11da0aa75574916fdb8fb343de2cde46a79b83b4